packages/next/test/fixtures/00-middleware/vercel.json

```diff
@@ -1,7 +1,42 @@
 {
   "version": 2,
-  "builds": [{ "src": "package.json", "use": "@vercel/next" }],
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@vercel/next"
+    }
+  ],
```

Comment:
You can remove `builds` now 😄

Reply:
This is still needed for the integration tests that run the builder manually I think since it uses the `runBuildLambda` util.